### DISCUSSION TITLE
Atomtype Q5 not found error -Go model simulation

### DISCUSSION THREAD

**Discussion by Asharuthra** [marrink-lab/vermouth-martinize#600] [General]
Hi, Sir/Ma'am,

I am newbie to cg martini simulation. I cant perform the go model simulation for my trans membrane protein  (human beta1 adrenergic receptor-7BU6.pdb), after lipid bilayer membrane insertion. I didn't face any error till membrane insertion.

I performed the minimisation after the lipid bilayer insertion with my trans-membrane protein. I am getting "atom type not found error" every-time. 

I didn't get any error after the minimisation of go model trans-membrane protein alone, but with lipid bilayer "atom type not found error" occurred. I tried with both martini and martini beta version itp file. nothing worked out.

Can you suggest me idea how to solve this error?
I have included my files and command-line for your perusal
[transmembrane_protein.zip](https://github.com/marrink-lab/vermouth-martinize/files/15431048/transmembrane_protein.zip)
![commandline](https://github.com/marrink-lab/vermouth-martinize/assets/44369152/8c4f4bcd-17d9-485f-8269-6be4a493ccac)
![Error](https://github.com/marrink-lab/vermouth-martinize/assets/44369152/6422b53b-4cfc-4af4-b691-0ec332a73036)

Thank you in advance for your help.
.

**Comment by pckroon** [comment]
`martini_v3.0.b.3.2.itp` is missing a bunch of particle definitions when compared to `martini_v3.0.0.itp`.

But this is the wrong place to ask about this

**Comment by Asharuthra** [comment]
Hello, Sir.

Thanks for the reply.  I got atom type not found error, after I ran with
martini3 ( martini_v3.0.0.itp) also.




On Fri, May 24, 2024 at 3:32 AM Peter C Kroon ***@***.***>
wrote:

> martini_v3.0.b.3.2.itp is missing a bunch of particle definitions when
> compared to martini_v3.0.0.itp.
>
> But this is the wrong place to ask about this
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/marrink-lab/vermouth-martinize/issues/598#issuecomment-2129201789>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKSQKADCMESYNBWP5HAF6LLZD4JLBAVCNFSM6AAAAABIHGWUZ6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMRZGIYDCNZYHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Best regards,

Rudhra O,

PhD Scholar,

C/o Prof. Sanjeev Kumar Singh,

CADD & Molecular Modeling Lab,

Department of Bioinformatics,

Alagappa University,

Karaikudi - 630004,

Tamil Nadu, India.

**Comment by csbrasnett** [comment]
You're getting these errors because you're not defining everything in one place. When you make your Go model, you get one .top file, which has various `#include`s and `#define`s in. The system.top file made by insane doesn't have these. Once you add all the definitions there it should process the system correctly. But you'll need to add in the correct topology for cholesterol as well. There is a [martini 3 cholesterol](https://github.com/Martini-Force-Field-Initiative/M3-Sterol-Parameters/tree/main) model available, but I don't know if this is what insane has to make systems, so you'll need to check.

**Comment by Asharuthra** [comment]
Thanks a lot for your reply, sir. 
I have mentioned all the itp files in system topology file except for cholesterol. 
I will include this cholesterol itp file and let you know the outcome as soon as i perform simulation.

**Comment by Asharuthra** [comment]
Hi, sir
I got "atom-type q5 not found", after i mentioned the Cholesterol itp in topology file too.
I ran insane command-line again without cholesterol, same error received "atom-type q5 not found". 
The human beta1 adrenergic receptor (7BU6.pdb )has many missing residues. I used modeller to fill the missing residues to prevent index error while performing the go model simulation. Go model simulation went well, I am getting atom type not found error only after membrane insertion.
Kindly suggest me how to solve this error?

**Reply by csbrasnett** [reply]
as Peter said, Q5 is not defined in `martini_v3.0.b.3.2.itp`. You need to make sure you have `#include martini_v3.0.0.itp` in the actual .top file that you use that insane has made for you., which in this case is system.top.

You can see the difference in the commands you gave in the image above:

`gmx grompp -p t4l_only.top -f minimization.mdp -c t4l_box29.gro -o minimization-vac.tpr -r t4l_cg.gro`

runs fine because martinize has given you a t4l_only.top that includes definitions of the go model.

when you proceed to:

`gmx grompp -f minimization.mdp -c system-fix.gro -p system.top -o ions.tpr`

the system.top that insane makes doesn't have the same header information, so grompp cannot process your system.

**Comment by Asharuthra** [comment]
Hi, Sir
As I mentioned earlier, I tried go model simulation with both martini 3 and martini 3 beta itp file. I keep on getting atom type not found error.
I ran  go model simulation from scratch again, and what I observed was that my protein itp file was named as ardb1_1, ardb1_2, and so on. For this, I got an error "atom type ardb1_1 not found". 
Afterwards,  I removed underscore_1 from the ardb1. Kept only ardb1 in my protein itp . Again, I got an error like "atom type ardb1 not found".
In my system topology file also, I renamed the protein to ardb1. I still got the same error "atom type ardb1 not found". 
What should I do to solve this error?. I attached my file here for your perusal.
Thanks a lot for your prompt reply and support.
[ardb.zip](https://github.com/marrink-lab/vermouth-martinize/files/15436423/ardb.zip)
![ardb1error](https://github.com/marrink-lab/vermouth-martinize/assets/44369152/30a4bbc1-7294-432e-9701-f7d14507dcad)
![ardb1_1 error](https://github.com/marrink-lab/vermouth-martinize/assets/44369152/3e4232e5-57bf-4eed-ad23-639bad7dc4eb)

**Comment by csbrasnett** [comment]
The problem is not with whichever version of martini 3 you're using. The problem is with what else you include in your top file. The Go Martini model contains extra particle definitions that your topology file is missing. I would strongly advise reading the [preprint](https://www.biorxiv.org/content/10.1101/2024.04.15.589479v1) to understand how the model is constructed and doing the [Martini 3 protein tutorial](http://cgmartini.nl/index.php/2021-martini-online-workshop/tutorials/564-2-proteins-basic-and-martinize-2) for some worked examples of how the extra particle definitions are included.

**Comment by Asharuthra** [comment]
Thank you, I will do that, sir.

On Mon, May 27, 2024 at 1:30 PM Chris Brasnett ***@***.***>
wrote:

> The problem is not with whichever version of martini 3 you're using. The
> problem is with what else you include in your top file. The Go Martini
> model contains extra particle definitions that your topology file is
> missing. I would strongly advise reading the preprint
> <https://www.biorxiv.org/content/10.1101/2024.04.15.589479v1> to
> understand how the model is constructed and doing the Martini 3 protein
> tutorial
> <http://cgmartini.nl/index.php/2021-martini-online-workshop/tutorials/564-2-proteins-basic-and-martinize-2>
> for some worked examples of how the extra particle definitions are included.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/marrink-lab/vermouth-martinize/discussions/600#discussioncomment-9567458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKSQKAHN3DTNPSPWKVV3VFDZELR3HAVCNFSM6AAAAABIHMR7RGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TKNRXGQ2TQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <marrink-lab/vermouth-martinize/repo-discussions/600/comments/9567458@
> github.com>
>


-- 

Best regards,

Rudhra O,

PhD Scholar,

C/o Prof. Sanjeev Kumar Singh,

CADD & Molecular Modeling Lab,

Department of Bioinformatics,

Alagappa University,

Karaikudi - 630004,

Tamil Nadu, India.
